maia/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
```

Comment:
nit:  it probably should be `Bump maia to 0.2.1` in the commit summary (you had 2 commits with `maia-core` in the name)